experimental/add_to_app/README.md

```diff
@@ -5,50 +5,64 @@ and the latest version of CocoaPods. See the [README](../README.md) in the
 `experimental` directory for details.***
 
 This directory contains a bunch of Android and iOS projects that each import
-a standalone Flutter module called `example_module`.
+a standalone Flutter module called `flutter_module`.
 
 ## Goals for this sample
 
-* Show developers how to add Flutter to their existing applications.
+* Show developers how to add Flutter to their existing applications.install
```

Comment:
Doesn't seem intentional

Reply:
Thank god one of us is paying attention at least. Fixed. 😄